sgf-viewer/src/content.ts

```diff
@@ -800,45 +783,429 @@ export const pages: object =
+                    "In the first game, for example, we see a typical game unfold for the first 100+ moves. The victim takes an early lead, reaching 98% internal win probability by <a class='clickable' onclick='setMove(`dec23-vs-gift`, 24)'>move 24</a>, so the adversary is playing many moves that would normally be considered subpar. There is otherwise little overt evidence of anything suspicious happening at this stage. Gradually, we see the adversary playing some incursions into the victim’s territory, notably at moves <a class='clickable' onclick='setMove(`dec23-vs-gift`, 117)'>117</a> and <a class='clickable' onclick='setMove(`dec23-vs-gift`, 127)'>127</a>. With correct play, these and nearby stones will eventually be captured.",
+                    "At <a class='clickable' onclick='setMove(`dec23-vs-gift`, 206)'>move 206</a>, we see the first instance of the victim offering two stones, though in this position it is not yet dangerous. This happens repeatedly over the following moves, both at this location and on the left side. After <a class='clickable' onclick='setMove(`dec23-vs-gift`, 263)'>move 263</a> though, when the adversary forms an eye at the top, any further gifts will lead to the victim inadvertently resurrecting the adversary’s entire group. This happens at <a class='clickable' onclick='setMove(`dec23-vs-gift`, 344)'>move 344</a>. After the adversary captures two stones at <a class='clickable' onclick='setMove(`dec23-vs-gift`, 345)'>345</a>, the victim would like to recapture one stone but is barred by the positional superko rule. Thus, the adversary secures two eyes with <a class='clickable' onclick='setMove(`dec23-vs-gift`, 347)'>move 347</a>, saving its group that should have died. This is a significant reversal, though the victim still appears to be winning. But then, at <a class='clickable' onclick='setMove(`dec23-vs-gift`, 366)'>move 366</a>, the same scenario unfolds on the left side, tipping the scales decisively in favor of the adversary."
+                ]
+            },
+            {
+                "title": "Human replica of gift attack",
+                "dir_name": "dec23-vs-human-gift",
+                "paths": ["/shared/match/paper-robustness/website-games/human-gift-vs-b18-v1.sgf"],
+                "max_games": 10,
+                "adversary": "Kellin Pelrine",
+                "victim": "<code>dec23-victim</code>, 1 visit",
+                "description": [
```

Comment:
I made some suggestions in the GDoc you may want to port over: https://docs.google.com/document/d/1dHhpCaQoHXUEXoulG7PLuAEdQkchUlZnuGjjgI3tCNQ/edit#heading=h.suo6dcn8qh6f (sorry for editing in wrong place!)